### ISSUE TITLE
Slovenia national road shield turned into a white rectangle

### ISSUE THREAD

**Issue by 1ec5** [ZeLonewolf/openstreetmap-americana#95]
[Slovenia’s A1 motorway relation](https://www.openstreetmap.org/relation/59655) is tagged `network=SI:AC`, so it should show up as <img src="https://upload.wikimedia.org/wikipedia/commons/f/ff/Avtocesta_A1.svg" height="24" align="top" alt="A1">. Instead, it shows up as a generic white rectangle:

![generic](https://user-images.githubusercontent.com/1231218/151150465-62a64990-f0c0-43a1-a9a0-c21e048db35d.png)

@Mashin6 added a shield for the `SI:AC` network in 4a9c7a02a9f152def0c1215872aeb6fc00c5f50f, but it fell out of the branch for #72 at some point. [The image](https://github.com/ZeLonewolf/openstreetmap-americana/blob/c24e5e8974a40bb5a524d7e1633f0dd7fb27843c/style/icons/shield40_eu_sl_2.svg) is still present, but it’s unused. I wonder if this also accounts for the errors I see when zoomed out on Europe:

```
Uncaught RangeError: mismatched image size
    sa https://zelonewolf.github.io/openstreetmap-americana/js/maplibre-gl-custom-shields.js:31
    la https://zelonewolf.github.io/openstreetmap-americana/js/maplibre-gl-custom-shields.js:31
    clone https://zelonewolf.github.io/openstreetmap-americana/js/maplibre-gl-custom-shields.js:31
    getGlyphs https://zelonewolf.github.io/openstreetmap-americana/js/maplibre-gl-custom-shields.js:35
    asyncAll https://zelonewolf.github.io/openstreetmap-americana/js/maplibre-gl-custom-shields.js:31
    getGlyphs https://zelonewolf.github.io/openstreetmap-americana/js/maplibre-gl-custom-shields.js:35
    asyncAll https://zelonewolf.github.io/openstreetmap-americana/js/maplibre-gl-custom-shields.js:31
    asyncAll https://zelonewolf.github.io/openstreetmap-americana/js/maplibre-gl-custom-shields.js:31
    getGlyphs https://zelonewolf.github.io/openstreetmap-americana/js/maplibre-gl-custom-shields.js:35
    getGlyphs https://zelonewolf.github.io/openstreetmap-americana/js/maplibre-gl-custom-shields.js:35
    processTask https://zelonewolf.github.io/openstreetmap-americana/js/maplibre-gl-custom-shields.js:31
    receive https://zelonewolf.github.io/openstreetmap-americana/js/maplibre-gl-custom-shields.js:31
```

Maybe someone force-pushed a commit, clobbering someone else’s, or maybe a merge conflict was resolved badly. I don’t think it’s possible to tell exactly what happened because of all the squash-merging that took place. We’ll probably want to go through the other commits to catch anything else that needs to be reapplied.

/ref #69

**Comment by 1ec5** [comment]
(Incidentally, the ISO&nbsp;3166 code for Slovenia is SI. SL is for Sierra Leone.)

**Comment by Mashin6** [comment]
Same for Austria and Croatia.

**Comment by ZeLonewolf** [comment]
Note that the original padding sizes will have to be shrunk now that @zekefarwell got everything down to actual-pixels numbers.

**Comment by zekefarwell** [comment]
Looks like #72 was merged in with a squash commit.  Makes history hard to follow.

**Comment by ZeLonewolf** [comment]
> Looks like #72 was merged in with a squash commit. Makes history hard to follow.

My bad!  Next time we'll do a proper protected feature branch for long-lead OMT features.
